src/routes/github/create-branch/github-create-branch-options-get.ts

```diff
@@ -68,13 +62,9 @@ export const GithubCreateBranchOptionsGet = async (req: Request, res: Response,
 
 };
 
-const validateGitHubToken = async (jiraHost: string, githubToken: string, logger: Logger, gitHubAppId?: number) => {
-	const githubUrl = await getGitHubApiUrl(jiraHost, gitHubAppId, logger);
-	await axios.get(githubUrl, {
-		headers: {
-			Authorization: `Bearer ${githubToken}`
-		}
-	});
+const getJiraHostFromTenantUrl = (jiraHostParam) =>  {
+	const siteName = jiraHostParam?.substring(0, jiraHostParam?.indexOf("."));
```

Comment:
The format that tenant url get passed to us from jira is
<JIRAHOSTNAME>.<DOMAIN>